src/main/java/org/opentripplanner/middleware/trip_monitor/CheckMonitoredTrip.java

```diff
@@ -0,0 +1,176 @@
+package org.opentripplanner.middleware.trip_monitor;
+
+import org.opentripplanner.middleware.models.MonitoredTrip;
+import org.opentripplanner.middleware.models.OtpUser;
+import org.opentripplanner.middleware.otp.OtpDispatcher;
+import org.opentripplanner.middleware.otp.OtpDispatcherResponse;
+import org.opentripplanner.middleware.otp.response.Itinerary;
+import org.opentripplanner.middleware.otp.response.LocalizedAlert;
+import org.opentripplanner.middleware.otp.response.Response;
+import org.opentripplanner.middleware.persistence.Persistence;
+import org.opentripplanner.middleware.trip_monitor.jobs.TripMonitorNotification;
+import org.opentripplanner.middleware.utils.DateUtils;
+import org.opentripplanner.middleware.utils.NotificationUtils;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.time.LocalDate;
+import java.time.format.DateTimeParseException;
+import java.util.ArrayList;
+import java.util.List;
+
+public class CheckMonitoredTrip implements Runnable {
+    private static final Logger LOG = LoggerFactory.getLogger(CheckMonitoredTrip.class);
+    private final MonitoredTrip trip;
+    private OtpDispatcherResponse injectedOtpResponse;
```

Comment:
Add some comments about this method and please rename the method to include some kind of indication that this variable is only used during testing.

Reply:
Do you mean this field (and are you referring to `injectedOtpResponse`)? Or did you mean a method somewhere else?